api/pkg/cli/cmd/install/install.go

```diff
@@ -153,6 +154,15 @@ func (opts *options) run() error {
 		}
 	}
 
+	var deprecated bool
+	if opts.resource.GetAnnotations()[deprecationAnnotation] == "true" {
+		deprecated = true
+	}
+
+	if deprecated {
+		_ = printer.New(out).String("WARN: this version has been deprecated")
```

Comment:
`_ = printer.New(out).String("WARN: This version has been deprecated")`

Can you please do this nit change

Reply:
Should we also use a `bulb` 💡 or something while logging the warning ?
cc @sm43  @piyush-garg